sd/etcd/registrar.go

```diff
@@ -4,13 +4,19 @@ import (
 	etcd "github.com/coreos/etcd/client"
 
 	"github.com/go-kit/kit/log"
+	"time"
+)
+
+const (
+	MinHeartBeatTime = time.Millisecond * 500
 )
```

Comment:
This constant should either be made private (I think that's preferable) or given a doc comment.